adds endpoint to get a single book

Sm0k3s opened the pull request.

Sm0k3s committed:
adds endpoint to get a single book

clemwek reviewed (changes requested):
Create folder `tests` and start on your tests

meshnesh commented on app/__init__.py:
```diff
@@ -15,10 +15,24 @@
     return jsonify({'books': books})
 
+#gets a single book by id
+@app.route('/api/v1/books/<int:book_id>', methods=['GET'])
```
Add doc string explaining what the functionality needs to accomplish. Refer to this resource to understand docstrings http://www.pythonforbeginners.com/basics/python-docstrings

meshnesh reviewed (commented):
init file shouldn't have any code, move it to a separate file.
init is a python initialisation package.

Good implementation or implementation and module approach

Sm0k3s committed:
restructures folders and moves data to data python file

Sm0k3s committed:
adds some tests for the models

Sm0k3s committed:
adds more tests

Sm0k3s committed:
Hello books api

Api endpoints for hello books an online app that manages a library's day to day operation such as the addition of new books, borrowing of books

clemwek commented on app/__init__.py:
```diff
@@ -1,24 +1,43 @@
 
+"""gets all books"""
 @app.route('/api/v1/books')
 def get_books():
```
update this to get books from data.py

clemwek commented on app/__init__.py:
```diff
@@ -1,24 +1,43 @@
+"""gets all users"""
+@app.route('/api/v1/users')
+def get_users():
+    return jsonify({'users': users})
```
get users from data.py

clemwek reviewed (changes requested):
Add more tests for the endpoints that you have added

Sm0k3s committed:
access of apis to models

Sm0k3s committed:
slight improvement on apis

Sm0k3s committed:
fixes for apis

Sm0k3s committed:
updates methods for Books class
